README.md

```diff
@@ -1,4 +1,4 @@
-<img src="https://arviz-devs.github.io/arviz/_static/logo.png" height=100></img>
+<img src="https://github.com/arviz-devs/arviz_governance/blob/main/arviz_logos/ArviZ_fav.png" height=100></img>
```

Comment:
I'm not certain whether it's preferably to link to the `raw` subdomain directly or not. Either way, because they're on GitHub, we could link to the full logo and activate the right one depending on the theme setting.
```suggestion
<img src="https://raw.githubusercontent.com/arviz-devs/arviz_governance/main/arviz_logos/ArviZ.png#gh-light-mode-only" width=200></img>
<img src="https://raw.githubusercontent.com/arviz-devs/arviz_governance/main/arviz_logos/ArviZ_white.png#gh-dark-mode-only" width=200></img>
```

If we use the full logo with the name, then maybe we should not have the ArviZ heading below.